fs/operations/multithread.go

```diff
@@ -314,7 +314,7 @@ func (w *writerAtChunkWriter) WriteChunk(ctx context.Context, chunkNumber int, r
 	case *bufio.Writer:
 		er2 := w.Flush()
 		if er2 != nil {
-			return -1, fmt.Errorf("multi-thread copy: flush failed: %w", err)
+			return -1, fmt.Errorf("multi-thread copy: flush failed: %w", er2)
```

Comment:
On the other hand; there was no reason for the existing code to create a separate error variable, I think.

So an alternative (better?) fix is this:

```go
err = w.Flush()
if err != nil {
	return -1, fmt.Errorf("multi-thread copy: flush failed: %w", err)
```

What do you think?

Reply:
yes, I think should use a easy way. I will update it